docs/DeveloperGuide.md

```diff
@@ -508,7 +509,8 @@ Use case ends.
 **Extensions**
 - 1a. The format of add method is invalid
     1a1. McGymmy shows an error message
-    Use case ends.
+    
+Use case ends.
```

Comment:
I think this line should be tabbed in, based on this example in the notebook
![image](https://user-images.githubusercontent.com/58925632/98463819-3eda4f80-21f9-11eb-80f2-533d8eb5c7ce.png)

Reply:
I dont think the example needs to followed religiously, i believe just having it standardised is sufficient